hibernate-core/src/main/java/org/hibernate/event/spi/EventManager.java

```diff
@@ -0,0 +1,176 @@
+/*
+ * Hibernate, Relational Persistence for Idiomatic Java
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later.
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.event.spi;
+
+import org.hibernate.Incubating;
+import org.hibernate.cache.spi.Region;
+import org.hibernate.cache.spi.access.CachedDomainDataAccess;
+import org.hibernate.engine.spi.EntityEntry;
+import org.hibernate.engine.spi.SharedSessionContractImplementor;
+import org.hibernate.persister.collection.CollectionPersister;
+import org.hibernate.persister.entity.EntityPersister;
+import org.hibernate.service.JavaServiceLoadable;
+
+@JavaServiceLoadable
```

Comment:
Since it's an SPI ... could you add some javadoc? Would be nice to at least write what it is, and if it's implementable by 3rd parties, what do they need to know?

Reply:
Added a short description, any suggestion is more than welcome :)